### ISSUE TITLE
Potentially confusing name of test service

### ISSUE THREAD

**Issue by chrisvfritz** [vuejs/vue-cli#876]
The name of `vue-cli-service test` seems a little misleading to me, since it's only running unit tests but the `test` directory contains both unit and e2e tests (assuming the user has chosen to install both).

What would you think about a rename to `vue-cli-service unit` to highlight the association with the folder?

**Comment by dennythecoder** [comment]
I agree with the intent, but it seems like there's a trend to put a verb in that place.  Perhaps `unit-test`?  Elongating it may be problematic, but it sounds better in my head

**Comment by dodekeract** [comment]
@dennythecoder Why can't we just change it to run both on `test`, and run unit on `test:unit` / e2e on `test:e2e`?

**Comment by dennythecoder** [comment]
@dodekeract, I dig that more than my suggestion.  :smiley:

**Comment by chrisvfritz** [comment]
I also like that. 🙂
